### ISSUE TITLE
some latitudes are zero when section is south of equator

### ISSUE THREAD

**Issue by StephenGriffies** [NOAA-GFDL/FMS#52]
In 

https://github.com/NOAA-GFDL/MOM6-examples/blob/dev/gfdl/ice_ocean_SIS2/OM4_025/diag_table.MOM6

we have two sections (below) that sample MOM fields completely south of the equator.  A handful of the southern latitudes have 0 for the latitude values, rather than -71 or the like. So the latitudes are not monotonic and that messes with the plotting, yielding spurious latitude values altogether for Ferret. 

We (Adrcroft and Griffies) think there is a bug in diag manager.

Here are the sections  

"ocean_model_z", "volcello", "volcello", "ocean_Drake_Passage",    "all", "mean", "-70. -70. -71. -54.5 -1 -1",2

"ocean_model_z", "volcello", "volcello", "ocean_Agulhas_section",  "all", "mean", "20. 20. -71.0 -34 -1 -1",2

An  example output is 

/archive/ogrp/CMIP6/OMIP/warsaw_201803_mom6_2018.04.06/OM4p25_IAF_fc0/gfdl.ncrc3-intel16-prod/pp/ocean_Drake_Passage/ts/monthly/20yr/ocean_Drake_Passage.198801-200712.umo.nc

**Comment by adcroft** [comment]
This problem seems to be a more a problem with mppnccombine. Example data in `/lustre/f1/Alistair.Adcroft/issue.mppnccombine` is causing all sorts of odd results at the cmobine stage: core dumps, changed values, non-monotonic coordinates. @Zhi-Liang

**Comment by Zhi-Liang** [comment]
I only have permission to access test.nc in /lustre/f1/Alistair.Adcroft/issue.mppnccombine. Could you change the permission of other files?

Thanks,

Zhi

**Comment by Zhi-Liang** [comment]
Hi,
   Through debugging, I found the issue might be from diag_manager regional output, not from mppnccombine. The file 19000101.ocean_Tropics.nc.0229 and 19000101.ocean_Tropics.nc.0245 both have an extra dimension and field yh_sub02. And yh_sub02 is same as yh_sub01. After I removed yh_sub02 from both files. mppnccombine runs fine and the output looks fine to me. The output is at /work/z1l/issue.mppnccombine/out.nc. I also noticed all the files have dimension and field xh_sub02, which is the same as xh_sub01. Though xh_sub02 does not affect combine, I think it is not needed. I will talk with our group to find someone to solve the issue in diag_manager.

Zhi
